help/stores-purchase/braintree.md

```diff
@@ -28,7 +28,7 @@ Go to [Braintree Payments][1] and sign up for an account.
 
    - In the _[!UICONTROL Merchant Location]_ section, verify that **[!UICONTROL Merchant Country]** is set to the location of your business.
 
-1. Under _[!UICONTROL Recommended Solutions]_, in the _[!UICONTROL Braintree Payments (by GENE Commerce v4.5.0)]_ section, click **[!UICONTROL Configure]**.
+1. Under _[!UICONTROL Recommended Solutions]_, in the _[!UICONTROL Braintree Payments (by [GENE Commerce](https://www.gene.co.uk/gene-braintree-payments/)) v4.6.1 - [Release Notes](https://support.gene.co.uk/support/solutions/articles/35000228529)]_ section, click **[!UICONTROL Configure]**.
```

Comment:
The following page 404s: https://support.gene.co.uk/support/solutions/articles/35000228529

Will it resolve on a future date closer to the 2.4.7 release?

Reply:
Yes, that page will be published on or before 2.4.7 release date.